packages/client/src/services/api/useAuthApi.ts

```diff
@@ -0,0 +1,44 @@
+import useBaseApi from './useBaseApi';
+
+export type AuthApiSignUp = {
```

Comment:
```suggestion
export type TAuthApiSignUp = {
```

типы обычно называют с префиксом `T` либо с постфиксом `Type` (я за `T`)

так читать код удобнее

Reply:
Done